assets/src/dashboard/components/modal/index.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import ReactModal from 'react-modal';
```

Comment:
For improved accessibility, need to use `Modal.setAppElement(appElement);`

You can basically copy/paste it from the editor:

https://github.com/google/web-stories-wp/blob/c6cf0f4bc324d70f43aa1ca43de98499ae8abf7a/assets/src/edit-story/index.js#L20-L48